website/docs/content/voice-and-tone/index.md

```diff
@@ -1,6 +1,30 @@
 ---
 title: Voice and Tone
-caption: Applications should be usable and accessible for all users of differing abilities.
-description: Applications should be usable and accessible for all users of differing abilities.
-previewImage: assets/illustrations/foundations/accessibility.jpg
+caption: How HashiCorp communicates with our users.
+description: How HashiCorp communicates with its users.
+previewImage: assets/illustrations/content/tone-and-voice.jpg
 ---
+
+“Voice and tone” is how we communicate with our users via copy. The “voice” is our personality and the “tone” is how we express that personality.
+
+## Voice
+
+The HashiCorp voice is smart, professional, technical, to the point, factual, clear, and helpful. Our goal is to be educational and helpful and not come across as know-it-alls, mean-spirited, or condescending.
+
+## Tone
+
+We're guiding our customers on a transformational journey to the cloud as educational stewards to illuminate their path to success. Our message should be smart, helpful, and to the point to enable human conversations.
```

Comment:
"To the point" and "helpful" are repeated here (helpful three times) which is slightly odd. Are there other words we could use there that make it less repetitive?